webplugin/template/mck-sidebox.html

```diff
@@ -1,953 +1,2267 @@
-<div id="mck-sidebox" class="mck-sidebox mck-box fade" role="main" aria-live="polite" style="visibility: hidden;">
-	<div class="mck-box-dialog mck-box-sm mck-right mck-bg-white">
-		<div id="mck-sidebox-content" class="mck-sidebox-content mck-box-content">
-			<div id="mck-conversation-header" class="mck-conversation-header mck-truncate n-vis"></div>
-			<div class="mck-box-top mck-title">
-				<div class="km-header-container">
-					<div class="km-conversation-header-title-container">
-						<div id="mck-tab-individual" class="mck-row n-vis">
-							<div class="mck-tab-link mck-back-btn-container">
-								<button id="mck-conversation-back-btn" aria-label="Back" role="button" tabindex="0" class="mck-conversation-tab-link">
-									<svg focusable="false" aria-hidden="true" class="mck-msg-list-icon" xmlns="http://www.w3.org/2000/svg" fill="#FFFFFF" height="30" viewBox="0 0 24 24" width="30">
-										<path d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z" />
-										<path d="M0-.5h24v24H0z" fill="none" />
-									</svg>
-
-								</button>
-							</div>
-							<div class=" mck-name-status-container mck-box-title mck-truncate">
-								<div>
-									<div class="mck-agent-image-container">
-										<div class="mck-agent-image-svg-container n-vis"></div>
-										<img class="mck-agent-image n-vis" src="">
-										<span class="mck-agent-status-indicator km-custom-widget-border-color n-vis"></span>
-									</div>
-								</div>
-								<div>
-									<div id="mck-tab-title" class="mck-tab-title mck-truncate mck-display-name">Conversations</div>
-									<div id="mck-tab-status" class="mck-tab-status mck-truncate n-vis force-n-vis"></div>
-									<div id="mck-agent-status-text" class="mck-agent-status-text  n-vis"></div>
-									<div class="mck-typing-box mck-truncate n-vis">
-										<span id="mck-typing-label">typing...</span>
-									</div>
-								</div>
-								
-							</div>
-						</div>
-						<div id="mck-tab-conversation" class="mck-row">
-							<h4 id="mck-conversation-title" class="mck-box-title mck-truncate">Conversations</h4>
-						</div>
-					</div>
+<div
```

Comment:
can that container changes in the html and other javascript break the UI customization that customers might have done at their end?